tdrs-backend/tdpservice/data_files/s3_client.py

```diff
@@ -33,5 +33,5 @@ def file_download(self, key, path, version_id):
         except ClientError as e:
             logger.error(e)
 
-        f = open(path, 'r')
+        f = open(path, 'rb')
```

Comment:
eric tested this in `raft` and noticed an issue still present (that differs from what i experience locally, though this issue is also present locally) - xslx files throw an exception as the s3 client attempts to open them with a utf-8 encoding. i pushed a fix to open all file downloads as binary, which, in my testing, fixes the download for xlsx files and allows other files (csv, txt) to be downloaded normally as well